src/printer-postcss.js

```diff
@@ -69,13 +69,19 @@ function genericPrint(path, options, print) {
         n.value.group.type === "value-value" &&
         n.value.group.group.type === "value-func" &&
         n.value.group.group.value === "extend";
+      const isComposed =
+        n.value.type === "value-root" &&
+        n.value.group.type === "value-value" &&
+        n.prop === "composes";
 
       return concat([
         n.raws.before.replace(/[\s;]/g, ""),
         n.prop,
         ":",
         isValueExtend ? "" : " ",
-        path.call(print, "value"),
+        isComposed
+          ? n.value.group.group.groups.join("")
+          : path.call(print, "value"),
```

Comment:
The doc at that point with all its definitions never contains a `type` of `line` or `if-break` to my knowledge. Hence, the `removeLines` would change much from what I can tell from reading here [here](https://github.com/tdeekens/prettier/blob/fix/css-modules-composes/src/doc-utils.js#L161). Is there another way of achieving the same or am I missing something?